assets/src/edit-story/components/panels/alignment/alignment.js

```diff
@@ -104,7 +105,20 @@ const alignmentButtonIds = [
 
 function ElementAlignmentPanel({ selectedElements, pushUpdate }) {
   const { isRTL } = useConfig();
-  const boundRect = getBoundRect(selectedElements);
+  const {
+    state: { pageSize },
+  } = useCanvas();
```

Comment:
I don't think you should be using the actual canvas size here, but the underlying data pixels.

You can get those from the `useUnits` hook and the units provider. Please CMIIW @dvoytenko.

The issue here is, that it doesn't actually work - setting to right or bottom aligned, doesn't actually move it to the right or bottom of the page:

![align-to-page-fails](https://user-images.githubusercontent.com/637548/79271304-94ba6500-7e6d-11ea-9d94-eb9603777cce.gif)

Reply:
Yes, the size should not be updated based on breakpoint. It should use actual width and height of the page.